pdq/cpp/hashing/pdqhashing.cpp

```diff
@@ -27,6 +26,37 @@ namespace facebook {
 namespace pdq {
 namespace hashing {
 
+namespace {
+
+// ----------------------------------------------------------------
+// Christoph Zauner 'Implementation and Benchmarking of Perceptual
```

Comment:
For fun, I found this paper and tried to figure out what this was referring to, and I can't find anything on the page that seems to match.